app/code/community/Oyst/OneClick/Helper/Data.php

```diff
@@ -173,4 +173,30 @@ public function isPaymentMethodOyst(Mage_Sales_Model_Order $order)
 
         return false !== strpos($order->getPayment()->getMethod(), $freepayPaymentMethod->getCode());
     }
+
+    /**
+     * Get metadata string.
+     *
+     * @return null|string
+     */
+    public function getTrackingMeta()
+    {
+        if (!Mage::getStoreConfig('oyst_oneclick/oneclick/tracking')) {
```

Comment:
Here you look for ```oyst_oneclick/oneclick/tracking```but it's ```oyst/oneclick/tracking```defined in ```system.xml```

Change value here.

![image](https://user-images.githubusercontent.com/861701/35229116-d910fefa-ff92-11e7-85f9-f36b4982c6ec.png)